pkgs/tools/security/gnupg/21.nix

```diff
@@ -29,7 +29,12 @@ stdenv.mkDerivation rec {
 
   NIX_LDFLAGS = stdenv.lib.optionalString stdenv.isDarwin "-lintl";
 
-  patches = [ ./fix-libusb-include-path.patch ];
+  patches = [
+    ./fix-libusb-include-path.patch
+    # TODO: Remove the following patches on the next gnupg release after 2.1.18
+    ./0028-scd-Backport-two-fixes-from-master.patch
+    ./0029-scd-Fix-use-case-of-PC-SC.patch
```

Comment:
Can we maybe fetch these patches from gnupg's version control system instead of adding them to nixpkgs?